frontend/src/services/RequestService.ts

```diff
@@ -19,5 +19,12 @@ export const loginRequest = (login: LoginData) =>
             }
         )
 
+export const fetch10Characters = () =>
+    axios.get("https://zoo-animal-api.herokuapp.com/animals/rand/10")
+        .then(res => res.data)
 
 
+export const fetch1Characters = () =>
```

Comment:
Don't know if you still need it, but if Gallery is the only component that calls this fetch, you can also delete the fetch here.